conf/battle/party.conf

```
@@ -54,8 +54,8 @@ idle_no_share: no
 party_even_share_bonus: 0
 
 // Display party name regardless if player is in a guild.
-// Official servers do not display party name unless the user is in a guild. (Note 1)
-display_party_name: no
+// Official servers display party name even if the user is not in a guild. (Note 1)
+display_party_name: yes
```

Assuming the old behaviour is pre-renewal
```suggestion
// Default on official servers: no for Pre-renewal, yes for Renewal. (Note 1)
//display_party_name: yes
```
in battle.cpp
```
#ifdef RENEWAL
	{ "display_party_name",                 &battle_config.display_party_name,              1,      0,      1,              },
#else
	{ "display_party_name",                 &battle_config.display_party_name,              0,      0,      1,              },
#endif
```

---

According to mrj, official pre-re behavior is to always display the party name, but there was a confusion because of a client bug that won't properly display it